platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/TESTS/compliance_attestation/test_a001/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../../ CACHE INTERNAL "")
+set(TEST_TARGET mbed-platform-feature-psa-compliance-attestation-a001)
+
+include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
+
+project(${TEST_TARGET})
+
+mbed_greentea_add_test(
+    TEST_NAME ${TEST_TARGET}
```

Nitpicking: 
```suggestion
    TEST_NAME
        ${TEST_TARGET}
```
to be consistent with the next few lines

---

Fixed for all test targets with multiple args for mbed_greentea_add_test()